config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1673,6 +1673,13 @@
 # Enable anonymous grading for assignments (default is false)
 # assignment.anon.grading.enabled=false
 
+# ######################################
+# SAK-29406 Allow Assignment tool to grade with two decimal points
+# ######################################
+# decimal points used to grade in new assignments (created after this code was applied), 
+# old assignments (created before this code) will continue using 1 decimal point to grade
+#assignment.grading.decimals=2
```

Comment:
One last thing here, you should mention what the default value will be if the property is not set. Ex:

`# DEFAULT: 2 (new assignments will use 2 decimal places for grading by default`

Then, change the example property to something other than the default value. Ex. change this:

`# assignment.grading.decimals=2`

To this:

`# assignment.grading.decimals=1`